### PR TITLE
Richard is back on twitter

### PR DESCRIPTION
**Pull request by richardjh**
I've been back on twitter since last November :)

### CODE REVIEW

**Comment by lboynton** [comment]
:+1: Look out world!

File: module/Phph/Site/view/phph/team/index.phtml [hunk 1]
```diff
@@ -24,7 +24,7 @@ $this->headTitle($title);
             <p>Richard is a Senior Developer with a background in Systems Administration. His main interests are in
                system and application security and is an advocate of frameworks and tools that reduce complexity and
                simplify development processes.</p>
-            <p><a href="http://richardjh.org/">richardjh.org</a></p>
+            <p><a href="http://richardjh.org/">richardjh.org</a> | <a href="https://twitter.com/richardjh_org">@richard_org</a></p>
```
Comment (lboynton) [inline module/Phph/Site/view/phph/team/index.phtml]: Hmm, shouldn't this be `@richardjh_org`?

**Comment by asgrim** [comment]
@lboynton quick on the merge button today, Travis hasn't even run yet :D

File: module/Phph/Site/view/phph/team/index.phtml (hunk 1 above)
Comment (richardjh) [inline module/Phph/Site/view/phph/team/index.phtml]: Yup. One minute.

**Comment by lboynton** [comment]
@asgrim I know, it was a bit risky with all those changes :D

**Pull request merged**